cal/cal/models.py

```diff
@@ -581,6 +580,9 @@ def query(self, calendar_ids=None, start=None, end=None):
         # Union over the querysets
         events_qs = reduce(lambda qs1, qs2: qs1 | qs2, querysets)
 
+        # Exclude future events
+        events_qs = events_qs.filter(start__lt=ensure_timezone_awareness(datetime.now()))
```

Comment:
We don't want to always exclude future events. There are instances where we need to set an end time in the future (see explanation in below comment)